packages/commerce-sdk-react/package.json

```diff
@@ -40,7 +40,7 @@
     "version": "node ./scripts/version.js"
   },
   "dependencies": {
-    "commerce-sdk-isomorphic": "^1.10.4",
+    "commerce-sdk-isomorphic": "/Users/vcua/Git/commerce-sdk-isomorphic",
```

Comment:
This PR needs the changes in https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/138 to work